static/js/reducers/dashboard.js

```diff
@@ -13,39 +14,48 @@ import {
   FETCH_PROCESSING,
   FETCH_SUCCESS,
 } from '../actions';
-import type { DashboardState } from '../flow/dashboardTypes';
+import type {
+  DashboardsState,
+  DashboardState,
+} from '../flow/dashboardTypes';
 import type { Action } from '../flow/reduxTypes';
 
-const INITIAL_DASHBOARD_STATE: DashboardState = {
+export const INITIAL_DASHBOARD_STATE: DashboardState = {
   programs: []
 };
 
-export const dashboard = (state: DashboardState = INITIAL_DASHBOARD_STATE, action: Action) => {
+const INITIAL_DASHBOARDS_STATE: DashboardsState = {};
+
+const updateDashboardState = (state, username, update) => (
+  _.merge({}, state, {[username]: update })
```

Comment:
Why are we using `_.merge` here? It would merge dashboard states together, but it seems like we just want to replace the old one with the new one if we have a newer version of a dashboard